src/main/java/com/github/cukespace/arquillian/asciidoctor/AsciidoctorObserver.java

```diff
@@ -326,8 +377,11 @@ private String get(final Map<String, String> extensionDef, final String key, fin
         return s == null ? val : s;
     }
 
-    private Logger getLogger() {
-        return Logger.getLogger(AsciidoctorObserver.class.getName());
+    /**
+     * mainly for testing purposes
```

Comment:
well I think package scope will not help because we will not be able to assign a value to _descriptorInstance_ in unit test, at least as easy as mocking the method. Also not makes sense  cause as per #4 we will not have unit tests on this.